MICS6_Master.do

```diff
@@ -76,6 +76,9 @@ foreach name in $newMICS6countries {
 	do "${DO}/4_sexual_health"
 
 * Housekeeping for women data
+	if inlist("`name'","Nepal2019") {
+	drop welevel2
```

Comment:
Could you clarify why dorp the variables here as we have the "keep" command at the very end that drop all the variables that are not needed?